### PR TITLE
Select all table entries with "CTRL+A"

### PR DESCRIPTION
**Pull request by melanieretter**
There is no right click context menu on tables anymore. Instead you can use the shortcut "CTRL+A" to select all table entries in a table.

### CODE REVIEW

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/smtable/SoftwareModuleTableLayout.java [hunk 1]
```diff
@@ -44,4 +45,42 @@
     void init() {
         super.init(smTableHeader, smTable, softwareModuleDetails);
     }
+
+    /*
```
Comment (SirWayne) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/smtable/SoftwareModuleTableLayout.java]: remove non-Javadoc

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/smtable/SoftwareModuleTableLayout.java [hunk 2]
```diff
@@ -44,4 +45,42 @@
...
+     * 
+     * @see org.eclipse.hawkbit.server.ui.common.table.AbstractTableLayout#
+     * isShortCutKeysRequired()
+     */
+    @Override
+    protected boolean isShortCutKeysRequired() {
+        return true;
+    }
+
+    /*
```
Comment (SirWayne) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/smtable/SoftwareModuleTableLayout.java]: remove  non-Javadoc

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/table/AbstractTable.java [hunk 3]
```diff
@@ -153,6 +153,14 @@ protected void selectRow() {
         }
     }
 
+    /**
+     * Select all rows in the table.
+     */
+    protected void selectAll() {
+        // only contains the ItemIds of the visible items in the table
```
Comment (SirWayne) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/table/AbstractTable.java]: only if multiselect is enabled

File: hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/TargetTableHeader.java [hunk 4]
```diff
@@ -402,20 +399,20 @@ private void addFilterTextField(final DistributionSetIdName distributionSetIdNam
         getFilterDroppedInfo().addComponent(filteredDistLabel);
         getFilterDroppedInfo().addComponent(filterLabelClose);
         getFilterDroppedInfo().setExpandRatio(filteredDistLabel, 1.0f);
```
Comment (Jonkno) [inline hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/management/targettable/TargetTableHeader.java]: Use F

**Comment by kaizimmerm** [comment]
Actually not working on mac as expected CMD-A. Can you have look at this?

**Pull request merged**